test/type/function_comment.stmt

```diff
@@ -0,0 +1,202 @@
+40 columns                              |
+>>> Block comment before first parameter.
```

Comment:
Do we have tests for inline block comments?
(And less exciting, but doc comments also)

Reply:
These ones are all inline block comments. Updated the descriptions to match and added tests of non-inline block and doc comments. (Deciding how comprehensive to be in tests is always a challenge.)